packages/govuk-frontend/src/govuk/components/accordion/accordion.mjs

```diff
@@ -140,7 +140,7 @@ export class Accordion extends GOVUKFrontendComponent {
     if (!$sections.length) {
       throw new ElementError(null, {
         componentName: 'Accordion',
-        identifier: `.${this.sectionClass}`
+        identifier: `Element (\`.${this.sectionClass}\`)`
```

Comment:
Wrinkle here: 

I've just gone with using `Element` any time the element in question is a `div`, but do we want to be specific? Or add a less generic qualifier?

For example, here would we prefer something like `Section element`? (Though not _that_, because `<section>` is a thing).

Reply:
Happy to drop "element" and just use the plain English name?

1. **Accordion:** Module (`[data-module=govuk-accordion]`)
1. **Accordion:** Section (`.govuk-accordion__section`)
1. **Accordion:** Section header (`.govuk-accordion__section-header`)
1. **Accordion:** Section heading (`.govuk-accordion__section-heading`)
1. **Accordion:** Section button (`.govuk-accordion__section-button`)
1. etc 

The word "Section" then doesn't have to mean `<section>` element